content/recipes/interoperability/c4c-clinical-trials.md

```diff
@@ -0,0 +1,302 @@
+(fcb-c4c-clinical-trials)=
+# Creating a metadata profile for clinical trial protocols
+
+
+
+
+````{panels_fairplus}
+:identifier_text: FCB084
+:identifier_link: 'https://w3id.org/faircookbook/FCB084'
+:difficulty_level: 3
+:recipe_type: applied_example
+:reading_time_minutes: 20
+:intended_audience: principal_investigator, data_manager, data_scientist, terminology_manager, ontologist
+:maturity_level: 3  
+:maturity_indicator: 19, 22
+:has_executable_code: nope
+:recipe_name: Creating a metadata profile for clinical trial protocols
+```` 
+
+
+## Main Objectives
+
+The purpose of this recipe is to describe the process to define and standardize study and protocol-level (meta)data commonly collected in paediatric clinical trials, with the aim of making trial data more Findable. The recipe details how to:
```

Comment:
"making trial more Finbable", there seems to be a part of "Interoperability" associated with the choice of ontologies (see comment later)